server/app/src/main/resources/indexer.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0"?>
```

Comment:
Yes, it uses for testing. The testing call to source class, I will try to move to test